packages/openapi-fetch/src/index.d.ts

```diff
@@ -15,6 +15,8 @@ import type {
 export interface ClientOptions extends Omit<RequestInit, "headers"> {
   /** set the common root URL for all API requests */
   baseUrl?: string;
+  /** custom dispatcher */
+  dispatcher?: unknown;
```

Comment:
We’ll want a more accurate type for this.

Reply:
Fixed.